commit 5803d31ae7704e0349821dcc20a94a1cddda69f1
Author: Tim Harder <radhermit@gmail.com>
Date:   Wed Sep 21 14:13:55 2011 -0700

    Fix build with libpng-1.5
    
    Explicitly include the zlib.h header for the Z_BEST_COMPRESSION and
    Z_DEFAULT_STRATEGY macros since >=libpng-1.5 doesn't pull it in anymore
    via the png.h header.

diff --git a/src/imageio/format/png.c b/src/imageio/format/png.c
index 40c7d48..c7be566 100644
--- a/src/imageio/format/png.c
+++ b/src/imageio/format/png.c
@@ -28,6 +28,7 @@
 #include <stdio.h>
 #include <png.h>
 #include <inttypes.h>
+#include <zlib.h>
 
 DT_MODULE(1)
 
